engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStore.scala

```diff
@@ -3,11 +3,15 @@ package cromwell.engine.workflow.lifecycle.execution
 import cromwell.backend.BackendJobDescriptorKey
 import cromwell.core.ExecutionStatus._
 import cromwell.core.{CallKey, JobKey}
+import cromwell.engine.workflow.lifecycle.execution.ExecutionStore.FqnIndex
 import cromwell.engine.workflow.lifecycle.execution.WorkflowExecutionActor.{apply => _, _}
 import wdl4s._
 
+import scala.language.postfixOps
 
 object ExecutionStore {
+  type FqnIndex = (String, Option[Int])
```

Comment:
I'd prefer to see case classes instead of `type` unless there's a good reason not to (e.g. "all of my fancy performance gains just went away")